src/training/ea/lipizzaner_trainer.py

```diff
@@ -188,6 +129,73 @@ def __init__(
         self._population_size = trainer_params.get("population_size", population_size)
         self._tournament_size = trainer_params.get("tournament_size", tournament_size)
 
+    #
+    # MIXTURE GENERATION
+    #
+
+    def optimize_generator_mixture(gans):
+        # TODO: implement
+        pass
+
+
+    #
+    # EVALUATION
+    #
+
+    def _generate_random_fitness_samples(self):
```

Comment:
Would be great if we could refactor this to be a standalone function in a helpers file